plugin/trino-postgresql/src/test/java/io/trino/plugin/postgresql/TestingPostgreSqlServer.java

```diff
@@ -146,4 +167,27 @@ public void close()
     {
         dockerContainer.close();
     }
+
+    public static class DatabaseEvents
+    {
+        private final TestingPostgreSqlServer server;
+        private final int currentEventsCount;
+
+        private DatabaseEvents(TestingPostgreSqlServer server)
+        {
+            this.server = requireNonNull(server, "server is null");
+            currentEventsCount = server.getRemoteDatabaseEvents().size();
+        }
+
+        public Stream<String> streamQueriesContaining(String queryPart, String... alternativeQueryParts)
+        {
+            ImmutableList<String> queryParts = ImmutableList.<String>builder()
+                    .add(queryPart)
+                    .addAll(ImmutableList.copyOf(alternativeQueryParts))
+                    .build();
+            return server.getRemoteDatabaseEvents().stream().skip(currentEventsCount)
```

Comment:
nit please put each chained method in separate line
```
server.getRemoteDatabaseEvents()
   .stream()
   .skip(currentEventsCount)
   .map()
   ...
```

Reply:
ok